core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -79,8 +106,12 @@ angular.module('oppia').controller(
         $scope.lastSuggestionToReview = remainingSuggestions.length <= 0;
         $scope.translationHtml = (
           $scope.activeSuggestion.change.translation_html);
+        $scope.status = $scope.activeSuggestion.status;
         $scope.contentHtml = (
           $scope.activeSuggestion.change.content_html);
+        $scope.editedContent = {
```

Comment:
I tried adding `editedTranslationHtml` instead of using `editedContent.html`, this time arround no errors were shown. But the new string value, which is set in the `schema-based-editor` for `editedTranslationHtml`  is not retrieved `$scope.updateSuggestion`. This was never the case when the html string value is passed as an object. Perhaps @sagangwee can help me offline regarding this.

Reference: https://github.com/oppia/oppia/pull/12424/files#r607285735
/cc @seanlip

Reply:
Ah, I see, I understand the issue. Just leave a comment above this one to explain that we're using an object here so that we can pass its properties by reference into the schema-based editor. Thanks for investigating.

(In general, if you do something in the codebase that might "look weird" to an outside observer but has a good reason behind it, just explain it in code comments.)